tests/unit/test_charm.py

```diff
@@ -98,9 +98,22 @@ def lightkube_client():
         yield client
```

Comment:
setting autouse=true on this lightkube client fixture is probably a good idea so that unit tests dont accidently try to call out via lightkube if the fixture is not used explicitly.